sdf/1.8/link.sdf

```diff
@@ -46,5 +46,6 @@
   <include filename="audio_source.sdf" required="*"/>
   <include filename="battery.sdf" required="*"/>
   <include filename="light.sdf" required="*"/>
+  <include filename="particle_emitter.sdf" required="*"/>
```

Comment:
I think `particle_emitter.sdf` is needed in `sdf/1.8/`.

Reply:
Added in 05545bd7099ba9b9a56cc357fee7758297d72629